playwright.config.ts

```diff
@@ -33,12 +33,11 @@ export default defineConfig<PluginOptions>({
 
 
     /* Collect trace when retrying the failed test. See https://playwright.dev/docs/trace-viewer */
-    trace: 'on-first-retry',
-
+    trace: 'on',
```

Comment:
`on-first-retry` makes the initial execution quite a bit faster, and less resource intensive